openfisca_country_template/tests/situations/a_single_parent_with_a_child.yaml

```diff
@@ -0,0 +1,21 @@
+- name: "A single parent with a child situation"
+  description: A simple situation just to test
+  period: 2017-01
+  absolute_error_margin: 0
+  households:
+    - parents: ["parent1"]
+      children: ["child1"]
+      accommodation_size: 100
+  persons:
+   - id: "parent1"
+     birth: '1961-01-15'
+     salary:
+      2017-01: 2400
+   - id: "child1"
+     birth: '2002-01-15'
```

Comment:
As we want to give best practice examples, it seems important to me that we use a coherent style of quotes. If possible, no quotes at all would be the best if YAML does support it 🙂